### DISCUSSION TITLE
PR-Flag: keep-branch

### DISCUSSION THREAD

**Discussion by emkZero** [importPI19fromDHGE/dhge-pi19-sem4#77] [Ideas]
Da ja nicht immer der Autor eine Branch mergt, wollen wir vielleicht eine Flag hinzufügen, die ausdrückt, dass eine Branch nach dem merge nicht gelöscht werden soll?
Der Default-Fall wäre dann löschen, was m.E. die Praxis ganz gut abbildet

**Comment by jsuesse** [comment]
Wie wäre die Implementierung? Label bei PR?

Übrigens sieht man die Diskussionen nicht, außer man kommt spontan vorbei

**Reply by Steve2955** [reply]
> Übrigens sieht man die Diskussionen nicht, außer man kommt spontan vorbei

Wenn man die Benachrichtigungen für das gesamte Repo aktiviert, sollte man eine Benachrichtigung bekommen

**Reply by jsuesse** [reply]
ja gut haha

Ich meinte viel mehr die (1) wie bei PR und Issues in der Leiste - gibt es nicht.

**Reply by RvNovae** [reply]
Ich hab die Diskussion bisher auch gar nicht gesehen 😓 
Schade, dass für den Tab keine Badge für "offene Dikussionen" angezeigt wird. Naja, kommt vielleicht nocht.

**Comment by Steve2955** [comment]
Können wir mMn gerne so implementieren, ist ja kein großer Aufwand, muss nur einmal bei allen ankommen 👍

**Reply by RvNovae** [reply]
Dem würd ich mich anschließen.

**Reply by emkZero** [reply]
Cool ^^ Können ja ein Broadcast auf Telegram mchen

**Comment by emkZero** [comment]
![image](https://user-images.githubusercontent.com/26359277/120984519-042e9a80-c77b-11eb-8e9f-82307809e30a.png)
